yaksh/templates/yaksh/viewcourse.html

```diff
@@ -0,0 +1,51 @@
+{% extends "manage.html" %}
+
+{% block title %} Courses {% endblock title %}
+
+{% block subtitle %} Courses {% endblock %}
+
+{% block css %}
+<link rel="stylesheet" media="all" type="text/css" href="{{ URL_ROOT }}/static/yaksh/css/course.css" />
+{% endblock %}
+
+{% block manage %}
+<center><h3> Course(s) Allotted </h3></center><br>
+{% if not courses %}
+    <center><h3> You are not added to any course(s) </h3></center>
+{% else %}
+{% for course in courses %}
+<h4>{{ course.creator.get_full_name }} added you to this course</h4>
+<div class="row show-grid">
+        <div class="span14">
+            <div class="row">
+                <div class="span6">
+                    <p>
+                        <b><u>Course</u></b>
+                        {% if course.active %}
+                            <span class="label success">Active</span>
+                        {% else %}
+                            <span class="label important">Closed</span>
+                        {% endif %}
+                    </p>
+                    <a href="{{URL_ROOT}}/exam/manage/course_detail/{{ course.id }}/">{{ course.name }}</a>
+                    </br>
+                </div>
+                <div class="span6">
+                    <p><b><u>Quiz(zes)</u></b></p>
+                    {% if course.get_quizzes %}
+                        {% for quiz in course.get_quizzes %}
+                            <a href="{{URL_ROOT}}/exam/manage/addquiz/{{ quiz.id }}/">{{ quiz.description }}</a><br>
+                    {% endfor %}
+                    {% else %}
+                        <p><b>No quiz </b>
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+    </div>
+    <br><br> 
+{% endfor %}
+<button class="btn primary" type="button" onClick='location.replace("{{URL_ROOT}}/exam/manage/add_course");'>Add New Course</button>

```

Comment:
Add quiz or course buttons not needed in the view allotted courses. Anyway we display add course button on the moderator home page or on list of courses.